exporter_test.go

```diff
@@ -492,7 +585,8 @@ type = "Apache-2.0"
       "command": "/worker/command",
       "args": ["worker", "command", "args"]
     }
-  ]
+  ],
+  "slices": null
```

Comment:
I am not sure we should add slices to this label. This information would be more useful paired with the sha in the array of slice layers in `io.buildpacks.lifecycle.metadata`. (adding to that label doesn't have to be a requirement right now)